gradle.properties

```diff
@@ -1,13 +1,80 @@
-# Mod properties
-modid=mixingasm
-archives_base=mixingasm
-# http://maven.apache.org/guides/mini/guide-naming-conventions.html
-group=makamys.mixingasm
-
-# Forge properties
-minecraft_version=1.7.10
-forge_version=10.13.4.1614-1.7.10
-update_url=https://raw.githubusercontent.com/makamys/Mixingasm/master/updatejson/update.json
-
-enable_mixin=false
-multiproject_structure=false
+modName = MixingAsm
+
+# This is a case-sensitive string to identify your mod. Convention is to use lower case.
+modId = mixingasm
+
+modGroup = makamys.mixingasm
+
+# WHY is there no version field?
+# The build script relies on git to provide a version via tags. It is super easy and will enable you to always know the
+# code base or your binary. Check out this tutorial: https://blog.mattclemente.com/2017/10/13/versioning-with-git-tags/
+
+# Will update your build.gradle automatically whenever an update is available
+autoUpdateBuildScript = false
+
+minecraftVersion = 1.7.10
+forgeVersion = 10.13.4.1614
+
+# Specify a MCP channel and mappings version for dependency deobfuscation and the deobfParams task.
+channel = stable
+mappingsVersion = 12
+
+# Define other MCP mappings for dependency deobfuscation
+remoteMappings = https://raw.githubusercontent.com/MinecraftForge/FML/1.7.10/conf/
+
+# Select a username for testing your mod with breakpoints. You may leave this empty for a random username each time you
+# restart Minecraft in development. Choose this dependent on your mod:
+# Do you need consistent player progressing (for example Thaumcraft)? -> Select a name
+# Do you need to test how your custom blocks interacts with a player that is not the owner? -> leave name empty
+developmentEnvironmentUserName = Developer
+
+# Define a source file of your project with:
+# public static final String VERSION = "GRADLETOKEN_VERSION";
+# The string's content will be replaced with your mod's version when compiled. You should use this to specify your mod's
+# version in @Mod([...], version = VERSION, [...])
+# Leave these properties empty to skip individual token replacements
+replaceGradleTokenInFile = Mixingasm.java
+gradleTokenModId = GRADLETOKEN_MODID
+gradleTokenModName = GRADLETOKEN_MODNAME
+gradleTokenVersion = GRADLETOKEN_VERSION
+gradleTokenGroupName = GRADLETOKEN_GROUPNAME
```

Comment:
`GRADLETOKEN_MODNAME` and `GRADLETOKEN_GROUPNAME` are unused, please leave the properties empty. Also, I'd like to see `GRADLETOKEN_MODID` not used ever, I think it has no benefit but rather adds an aditional chance to accidentally change the modid, either because someone accidentally changes it or because Gradle fails to replace the token.

Reply:
All properties are set by default now in example mod for some reason, should we really leave them empty?

Reply:
The examplemod comes with some example classes where they are used. These `gradleToken*` fields are only used so that `build.gradle` knows which strings to replace. If the string is empty, it does not attempt to find & replace it.

Reply:
Yeah but I still can't see the issue with that. I have removed them though.